spec/spec_helper.rb

```diff
@@ -10,6 +10,7 @@
 require_relative 'support/http'
 require_relative 'support/file'
 require_relative 'support/manifest'
+require_relative 'support/cf.rb'
```

Comment:
Is this helper really required in all specs? If not, I'd prefer to require the file only where it is needed.